### PR TITLE
added template documentation for issues and pull requests

### LINKED ISSUE #91: Set up Template Documentation for Issues and Pull Requests
As discussed we'll be trying out a template for both Issues and Pull requests going forward.

This is a github functionality that will recognise .md files provided with the right naming convention and populate new Issue or PR windows with the template stored in our repo to help us make all submissions uniform going forward.

The following file structure is to be introduced:

.github/
|__ISSUE_TEMPLATE.md
|__PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by camelPhonso**
closes #91

Added `ISSUE_TEMPLATE.md` and `PULL_REQUEST_TEMPLATE.md` with suggestion text.

I'd suggest viewing these in the formatted markdown file. Request any changes needed - these will populate our PR and Issue description boxes automatically once merged.

### CODE REVIEW

**Comment by eliazzo** [comment]
This looks great! Love the checklist ✅ Maybe I'm being an idiot but I don't really get the Steps to Reproduce part? What is this for?

```## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. step 1
2. step 2
3. you get it...
```

**Comment by camelPhonso** [comment]
> This looks great! Love the checklist ✅ Maybe I'm being an idiot but I don't really get the Steps to Reproduce part? What is this for?
> 
> ```
> 
> Please provide detailed steps for reproducing the issue.
> 
> 1. step 1
> 2. step 2
> 3. you get it...
> ```

Everything below line 18 is for reporting a bug. So it's the steps to reproduce the error so others can debug it properly

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Expected Behaviour
+
+Please describe the behaviour you are expecting
+
+# Current Behaviour
+
+What is the current behaviour?
+
+<hr>
+# Failure Information (for bugs 🪲)
```
Comment (nichgalzin) [inline .github/ISSUE_TEMPLATE.md:19]: Is this a header? Not displaying on my as such in the preview mode.

**Review by eliazzo — APPROVED** [review]
👌

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Checklist:
+
```
Comment (nichgalzin) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: I like this a lot! Think it looks great!

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (camelPhonso) [reply .github/ISSUE_TEMPLATE.md]: It was! Thanks for spotting that.

I needed to add a line between the <hr> and the header for markup to pick up.

Took the chance to make this section a bit clearer by separating the message on reporting bugs. Hopefully the guidance on filling these out is easier to follow now

**Pull request merged**